### ISSUE TITLE
Top Toolbar (tool presets) Not showing up

### ISSUE THREAD

**Issue by SpitfireStoryboards** [Pullusb/storytools#62]
Even when switched on in pref, no top toolbar shows up
<img width="1280" alt="ScreenHunter 13142" src="https://github.com/user-attachments/assets/339cc2b2-2f9e-4706-8a53-683cd3991d28">

4.2.2 addon 1.92.0

**Comment by SpitfireStoryboards** [comment]
happens on 4.0 too

**Comment by Pullusb** [comment]
Hi sorry for the bad communication on my side.
The new version are for blender 4.3.

For 4.2 and less, the last compatible version is available in the relases page: https://github.com/Pullusb/storytools/releases

**Comment by SpitfireStoryboards** [comment]
Oh! Thanks



________________________________

**Comment by SpitfireStoryboards** [comment]
No problem. Was keen to try it out.

________________________________

**Comment by Pullusb** [comment]
I relealized the way github packs the release might create addon duplication in Blender (as it adds the version in the addon folder name).
So I just added a 'storytools.zip' to the release page and updated readme's download link for 4.2.
Use this link to use in 4.2.


If you already used previous one, you might want to check in your addon folder to delete any `storytools-1.19.6` (renaming it to `storytools` is fine)
